addons/website/static/src/js/menu/edit.js

```diff
@@ -96,6 +101,24 @@ var EditPageMenu = websiteNavbarData.WebsiteNavbarActionWidget.extend({
             $target: ev.data.$target,
         });
     },
+    /**
+     * Set editor message when editable area is empty
+     * We need translated message otherwise it can be done with LESS
+     *
+     * @private
+     */
+    _onEmptyEditableArea: function () {
+        var $wrapwrap = $('#wrapwrap'); // TODO find this element another way
+        $wrapwrap.find('.oe_structure.oe_empty:empty, [data-oe-type="html"]:empty').attr('data-editor-message', _t('DRAG BUILDING BLOCKS HERE'));
+    },
```

Comment:
This should not be done multiple times, only when the editor is first launched (and it should not target `:empty` elements only). So this does not need to be in a function at all.